pkg/ansible/watches/watches.go

```diff
@@ -141,9 +142,45 @@ func (w *Watch) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	w.Finalizer = tmp.Finalizer
 	w.AnsibleVerbosity = getAnsibleVerbosity(gvk, ansibleVerbosityDefault)
 	w.Blacklist = tmp.Blacklist
+	w.addRolePlaybookPaths()
+
 	return nil
 }
 
+// addRolePlaybookPaths will add the full path based on the current dir
+func (w *Watch) addRolePlaybookPaths() {
+	w.Playbook = getFullPath(w.Playbook)
+	w.Role = getFullRolePath(w.Role)
+	if w.Finalizer != nil && len(w.Finalizer.Role) > 0 {
+		w.Finalizer.Role = getFullRolePath(w.Finalizer.Role)
+	}
+	if w.Finalizer != nil && len(w.Finalizer.Playbook) > 0 {
+		w.Finalizer.Playbook = getFullPath(w.Finalizer.Playbook)
+	}
+}
+
+// getFullPath will return a valid full path for the playbook
+func getFullPath(path string) string {
+	if len(path) > 0 && !filepath.IsAbs(path) {
+		return filepath.Join(projutil.MustGetwd(), path)
+	}
+	return path
+}
+
+// getFullRolePath will return a valid full path for the role
+func getFullRolePath(path string) string {
+	if len(path) > 0 && !filepath.IsAbs(path) {
+		envVar, ok := os.LookupEnv("ANSIBLE_ROLES_PATH")
+		if ok && len(envVar) > 0 {
+			return filepath.Join(envVar, path)
+		}
```

Comment:
Don't we still need to handle the case of a multiple paths in the ANSIBLE_ROLES_PATH value? (e.g. `path1/roles:path2/roles:/absolute/path3/roles`)

Reply:
Done 👍